bin/quipucords-installer

```diff
@@ -120,14 +120,34 @@ main() {
     esac
   done
 
-  if [[ $# -ne 1 ]]; then
-    echo "Error: Exactly one command is required." 1>&2
```

Comment:
we might need to adjust this check per command. Only upgrade accepts other arguments now.

or maybe we keep it as is and document that other args will be ignored?

Reply:
I also don't love this, but I'm ignoring it for the time being because I _hope_ to get back to the ctl where each command will actually own its own argument validation. I think this is fine for the short term.